.github/scripts/build.functions.sh

```diff
@@ -19,7 +19,31 @@ function should_build_oss() {
 function should_build_ee() {
 
   local release_type=$1
-  if [[ $release_type == "ALL" || $release_type == "EE" ]]; then
+  if [[ $release_type =~ ^(ALL|EE)$ ]]; then
+    echo "yes"
+  else
+    echo "no"
+  fi
+}
+
+# Checks if we should rebuild the docker EE image description.
+# Returns "yes" if we should rebuild it or "no" if we shouldn't.
+function should_build_readme_ee() {
```

Comment:
We already have `should_build_oss` and `should_build_ee` which does the same check apart from `README` 

Could be not add `README` and call the same above functions and remove `*readme*` functions to avoid duplication?

Reply:
I don't understand, 

the logic is as follows:
1. If EE image needs to be rebuilt -> rebuild EE image and update readme on dockerhub
2. If OSS image needs to be rebuilt -> rebuild OSS image and update readme on dockerhub
3. If only README needs to be rebuilt -> update readme on dockerhub

How would you like to accomplish 3rd case without `should_build_readme_ee/should_build_readme_oss` functions?

Reply:
Sorry. Ignore this. Thought I didn't submit the comment but clearly I did!